README.md

```diff
@@ -0,0 +1,186 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#Hello-microverse)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upkill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To run this application localy, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-An IDE (Intergrated Development Environment) such vscode, atom etc ...
+<br>
+-web browser such as chrome, firefox etc ..
+
+### Setup
+
+To clone this project, run the following commands::<br>
+cd your-prefered-folder-name :<br>
+
+git clone https://github.com/migo1/Hello-Microverse.git<br>
+
+### Usage
+
+To run this app in your browser, run in your terminal:
+`cd Hello-Microverse`
+
+Then open the live server or open the `index.html` file with your browser
+## Install Linters
+
+You can find linters for most of the programming languages, e.g. Rubocop for Ruby or ESLint for JavaScript.
+
+Also, there are many ways you can integrate a linter in your workflow:
+
+-text editor plugin<br>
+-GitHub Actions<br>
+-GitHub apps<br>
+
+## Set up Linters
+
+**Note:** The npm package manager is going to create a node_modules directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+
+# .gitignore
+
+node_modules/
+
+## Web Hint
+
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+**NOTE:** If you are using Windows, make sure you initialize npm to create `package.json` file.
+
```

Comment:
Good job by setting up correctly linters for this project and by understanding what is it for but the visitors don't need to redo the work they already have gitignore and all linter files you can mention only that they need to run npm install to make it work

Reply:
Noted

Reply:
@souad988 I've corrected the redundancy on the readme file